10300.xml

```diff
@@ -0,0 +1,83 @@
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>LWM2M Meta Object</Name>
+    <Description1><![CDATA[Use this meta object to provide one or more locations for XML files defining vendor-specific objects. ]]></Description1>
+    <ObjectID>10300</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10300</ObjectURN>
+    <LWM2MVersion>1.0</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Multiple</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+      <Item ID="0">
+        <Name>ObjectID</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>
+        </RangeEnumeration>
+        <Units>
+        </Units>
+        <Description><![CDATA[Object ID of the vendor-specific object]]></Description>
+      </Item>
+      <Item ID="1">
+        <Name>ObjectURN</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>
+        </RangeEnumeration>
+        <Units>
+        </Units>
+        <Description><![CDATA[Object URN of the vendor-specific object]]></Description>
+      </Item>
+      <Item ID="4000">
+        <Name>ObjectInstanceHandle</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Objlnk</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units></Units>
+        <Description><![CDATA[Reusable LWM2M resource. This can be used to differentiate object instances having the same object ID and URN (for example, for Object IDs in the range 26241 - 32768)]]></Description>
+      </Item>
+      <Item ID="2">
+        <Name>MD5</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>
+        </RangeEnumeration>
+        <Units>
+        </Units>
+        <Description><![CDATA[MD5 value of the XML file in the URL(s)]]></Description>

```

Comment:
Would more modern algorithm make sense since MD5 is broken as crypto hash? Should there be crypto agility, e.g., with another resource to define what is the algorithm used, and a default algo if that is missing?

Reply:
@bsilverajan I suggest to continue using this PR, there is no need fo resubmit it now that a specific branch and PR has been created.
Feel free to update this PR with the changes that you suggest, we will change the name in the registry file to Tampere University, TU.